internal/openapi-generator/api/openapi.yaml

```diff
@@ -9418,9 +9420,8 @@ components:
         in the type URL, for example "foo.bar.com/x/y.z" will yield type
         name "y.z".
 
-
         JSON
-
+        ====
```

Comment:
Hmm, this is surprising.  There wasn't a merge conflict here was there?

Reply:
That is in and seems to come from openapi.json. That big blob of text seems to come from the protobuf description of `Any`, if I'm reading it right -- probably a change in protobuf library version introduced this small docs change? In any event, I don't think it's due to a merge conflict or anything like that.